plugins/outputs/sql/sql.go

```diff
@@ -158,6 +165,14 @@ var sampleConfig = `
   #  timestamp            = "TIMESTAMP"
   #  defaultvalue         = "TEXT"
   #  unsigned             = "UNSIGNED"
+  #  bool                 = "BOOL"
+
+  ## This setting controls the behavior of the unsigned value. By default the
+  ## setting will take the integer value and append "unsigned" to it. The other
```

Comment:
This doesn't match the readme now.

```suggestion
  ## setting will take the integer value and append the unsigned value to it. The other
```